gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -363,27 +367,31 @@ func (p *glusterBlockProvisioner) Delete(volume *v1.PersistentVolume) error {
 			glog.Errorf("glusterblock: error [%v] when running command %v", cmdErr, deleteCmd)
 			return cmdErr
 		}
-		glog.V(1).Infof("gluster-block: Successfully deleted Volume %v ", delBlockVolName)
+		glog.V(1).Infof("glusterblock: successfully deleted Volume %v ", delBlockVolName)
 
 	case "heketi":
-		glog.V(1).Infof("heketik: Deleteing Volume %v", delBlockVolName)
+		glog.V(1).Infof("glusterblock: opmode[heketi]: deleting Volume %v", delBlockVolName)
 	default:
-		glog.Errorf("Unknown OpMode, failed to delete volume %v", delBlockVolName)
+		glog.Errorf("glusterblock: Unknown OpMode, failed to delete volume %v", delBlockVolName)
 
 	}
 
+	deleteSecErr := p.client.Core().Secrets(p.volConfig.BlockSecretNs).Delete(p.volConfig.BlockSecret, nil)
```

Comment:
@wongma7 its per PV and it holds the user credentials of that block volume. At time of PV deletion it should go away.